### PR TITLE
Table component

### LINKED ISSUE #71: Implement table component

### PR DESCRIPTION
**Pull request by bramvanhoutte**
Closes #71

### CODE REVIEW

File: src/components/Table/TableCell/TableCell.module.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+.cellAlignRight {
+  text-align: right;
+}
+
+.cellAlignCenter {
+  text-align: center;
+}
+
+.cell:hover ~ .cell {
```
Comment (RobinWijnant) [inline src/components/Table/TableCell/TableCell.module.css]: I just learned a new selector but I can't see the effect in the browser

File: src/components/Table/TableRow/TableRow.module.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.row:nth-of-type(even) td {
```
Comment (RobinWijnant) [inline src/components/Table/TableRow/TableRow.module.css:1]: You are styling elements not in this component.
Is it possible to leave td away?:
```css
.row:nth-of-type(even) {}
```

Comment (RobinWijnant) [reply src/components/Table/TableRow/TableRow.module.css:1]: You can set the background right away on the row:
https://www.w3schools.com/css/tryit.asp?filename=trycss_table_color

File: src/components/Table/TableBody/TableBody.tsx [hunk 3]
```diff
@@ -1,12 +1,22 @@
...
 
 const TableBody: React.FC<Props> = ({ className, children }: Props) => {
-  return <tbody className={className}>{children}</tbody>;
+  const cellContextValues: CellContext = {
+    location: 'body',
+  };
+
+  return (
+    <tbody className={className}>
+      <cellContext.Provider value={cellContextValues}>{children}</cellContext.Provider>
```
Comment (RobinWijnant) [inline src/components/Table/TableBody/TableBody.tsx:17]: A react component is pascal cased

Comment (bramvanhoutte) [reply src/components/Table/TableBody/TableBody.tsx:17]: It's not a react component, it's a context. But is just checked and the React documentation also pascal cases context, so will change.

File: src/components/Table/TableRow/TableRow.module.css (hunk 2 above)
Comment (bramvanhoutte) [reply src/components/Table/TableRow/TableRow.module.css:1]: Yes, but then I won't be able to a set a border radius.

**Pull request merged**